src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,30 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's block in the address book.
+ */
+
+public class Block {
+
+    private String block;
+
+    public Block(String block) {
+        this.block = block;
+    }
+
+    public String getBlock() {
```

Comment:
Is there a point to the getter function here if `toString()` also fulfils the same function? Same for `Street`, `Unit` and `Postal Code`